src/components/Countries.js

```diff
@@ -0,0 +1,60 @@
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import Card from "./Card";
+
+const Countries = () => {
+  const [data, setData] = useState([]);
+  const [rangeValue, setRangeValue] = useState(36);
+  const [selectedRadio, setSelectedRadio] = useState("");
+  const radios = ["Africa", "America", "Asia", "Europe", "Oceania"];
+
+  // le useEffect se joue lorsque le composant est monté
+  useEffect(() => {
+    axios
+      .get("https://restcountries.com/v3.1/all")
+      .then((res) => setData(res.data));
+  }, []);
+  return (
+    <div className="countries">
+      <ul className="radio-container">
+        <input
+          type="range"
+          min="1"
+          max="250"
+          defaultValue={rangeValue}
+          onChange={(e) => {
+            setRangeValue(e.target.value);
+          }}
+        />
+        {radios.map((continent) => (
+          <li>
+            <input
+              type="radio"
+              id={continent}
+              name="continentRadio"
+              checked={continent === selectedRadio}
+              onChange={(e) => setSelectedRadio(e.target.id)}
+            />
+            <label htmlFor={continent}>{continent}</label>
+          </li>
+        ))}
+      </ul>
+      {selectedRadio && (
+        <button onClick={() => setSelectedRadio("")}>
+          Annuler la recherche
+        </button>
```

Comment:
@MEDATTA0 
Make this a button component. The component should take the text, color, and function for onclick. 
This way you can reuse your component in the app anywhere needed. It is useful in case you are developing a system that has many buttons with the same style but different behavior. 


```suggestion
        <Button 
        text="Annuler la recherche" 
        color="red" 
        onClick={handleCancelClick} 
      />
```